terraform/containers.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "name": "gtsite-service",
+    "image": "fosskers/gtsite-service",
+    "memory": 1024,
+    "essential": true,
+    "portMappings": [
+      {
+        "containerPort": 9090,
+        "hostPort": 9090
+      }
+    ]
+  },
+  {
+    "name": "gtsite-nginx",
+    "image": "fosskers/gtsite-nginx",
+    "memory": 128,
+    "essential": true,
+    "links": [
+      "gtsite-service",
+      "gt-transit",
+      "gt-chatta-demo"
+    ],
+    "portMappings": [
+      {
+        "containerPort": 8080,
+        "hostPort": 8080
```

Comment:
`hostPort` here should be set to 0 so that ECS can handle dynamic port mappings at the target group.